client/source/components/RR/ReviewList/AddReview/StarRating.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaStar, FaRegStar } from 'react-icons/fa';
```

Comment:
if you want consistency with the AverageStars  you can import EmptyStar and FullStar here and simply change the names on lines 13 & 23 but totally optional and this looks great

Reply:
Good to know! I'll get it aligned once I start playing around the stars for a full sweep of consistency on the next PR